yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -182,7 +185,10 @@ func runExportDataStatusCmd() ([]*exportTableMigStatusOutputRow, error) {
 			partitions := strings.Join(partitions, ", ")
 			displayTableName = fmt.Sprintf("%s (%s)", displayTableName, partitions)
 		}
-		snapshotStatus, _ := exportedSnapshotStatus.Get(finalFullTableName)
+		snapshotStatus, ok := exportedSnapshotStatus.Get(finalFullTableName)
+		if !ok {
+			continue
```

Comment:
Should we display partial data? (i.e. status available only for a subset of rows?). It seems bad IMO.

I would be inclined to do the following: 
explicitly check in the beginning if we are able to get status (exportSnapshotStatus/export_status.json file is available). If not, then just return without printing anything.

Reply:
That check is already present above. When we read the files, we return from there if the file doesn't exist. This check I added is only applicable to the window in which the file is created but not completely populated with content, e.g., FilePath.